### PR TITLE
Add NativeModules and NativeEventEmitter

### PR DESCRIPTION
**Pull request by wokalski**

### CODE REVIEW

File: src/NativeEventEmitter.re [hunk 1]
```diff
@@ -0,0 +1,17 @@
+type t;
+
+type emitterSubscription;
+
+external create : NativeModules.t => t =
+  "NativeEventEmitter" [@@bs.new] [@@bs.module "react-native"];
+
+external addListener : t => string => (Js.Dict.t 'a => unit) => emitterSubscription = "" [@@bs.send];
```
Comment (saschatimme) [inline src/NativeEventEmitter.re]: Maybe `Js.t. {.} ` instead of `Js.Dict.t 'a`?

File: src/NativeModules.re [hunk 2]
```diff
@@ -0,0 +1,8 @@
+type t;
+
+external nativeModules : Js.Dict.t 'a = "NativeModules" [@@bs.module "react-native"];
```
Comment (saschatimme) [inline src/NativeModules.re]: gleiche wie oben

File: src/NativeEventEmitter.re (hunk 1 above)
Comment (wokalski) [inline src/NativeEventEmitter.re]: yeah, why not

Comment (wokalski) [inline src/NativeEventEmitter.re]: hm, no. It doesn't work. `Js.t` is different than `Js.Dict.t`.

Comment (saschatimme) [inline src/NativeEventEmitter.re]: Well, if you got something working with the API then we got with your version :)

**Pull request merged**